misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsMetrics.kt

```diff
@@ -9,60 +9,62 @@ internal class SqsMetrics @Inject internal constructor(metrics: Metrics) {
   val jobsEnqueued = metrics.counter(
       "jobs_enqueued_total",
       "total # of jobs sent to a queueName",
-      listOf("queueName")
+      // Duplicate labels so we don't break existing dashboards/detectors
+      // TODO(hamdan): remove queueName once no one's using it
+      listOf("queueName", "QueueName")
```

Comment:
don't you have to update the callers to provide both?

Reply:
Updated callers to provide both labels

Reply:
leave a comment we are using `QueueName` to be consistent with AWS SQS